src/states/DraggingState.ts

```diff
@@ -84,177 +82,54 @@ class DraggingState extends State {
      * |--------|--------------|
      * [][      |<-Anchor    ][] <- Panel + Half-Gap
      */
-    let minimumDistanceToChange = isNextDirection
-      ? currentPanel.getSize() - currentPanel.getRelativeAnchorPosition() + halfGap
-      : currentPanel.getRelativeAnchorPosition() + halfGap;
-    minimumDistanceToChange = Math.max(minimumDistanceToChange, options.threshold);
-
-    let duration = defaultDuration;
-    let panelToMove: Panel;
+    const currentPanelPosition = currentPanel.getPosition();
+    const currentPanelSize = currentPanel.getSize();
+
+    let minimumDistanceToChange: number;
```

Comment:
@WoodNeck 
Couldn't parts related to minimumDistanceToChange also enter the MoveType class area?

Reply:
It could be done, but I also thought that the only changing part will be only for `freeScroll` move type, so I might be hesitated.
But I guess you're right. This might be included on `MoveType` I guess.